editor/src/components/navigator/navigator-item/expandable-indicator.tsx

```diff
@@ -19,15 +19,18 @@ export const ExpandableIndicator: React.FunctionComponent<
 > = React.memo((props) => {
   return (
     <div data-testid={props.testId} style={{ width: 16, height: 16, ...props.style }}>
-      {props.visible ? (
-        <Icn
-          category='semantic'
-          type={`expansionarrow-${props.collapsed ? 'right' : 'down'}`}
-          color={props.selected ? 'on-highlight-main' : 'main'}
-          onMouseDown={props.onMouseDown}
-          onClick={props.onClick}
-        />
-      ) : null}
+      <Icn
+        category='semantic'
+        type={`expansionarrow-${props.collapsed ? 'right' : 'down'}`}
+        color={props.selected ? 'on-highlight-main' : 'main'}
+        style={{
+          pointerEvents: props.visible ? 'all' : 'none',
+          opacity: props.visible ? 1 : 0,
+          cursor: 'pointer',
```

Comment:
Remind me - does this not trigger on elements if opacity is zero, or do you need to use `visibility: hidden` or `display: none`?

Reply:
great idea, `opacity` does trigger, so `visibility` would be much better, added it